src/models/filter.rs

```diff
@@ -166,6 +208,16 @@ impl SourceCodeUnit {
             return false;
           }
         }
+        for query_with_holes in filter.contains() {
```

Comment:
```suggestion
               // <put a comment> 
                for query_with_holes in filter.contains() {
```

do something similar for `not-contains` loop too